### ISSUE TITLE
Not clear existing filter conditions in the Slow-Query page after entering the slow-query detail page

### ISSUE THREAD

**Issue by qw4990** [pingcap/tidb-dashboard#1466]
## Feature Request

**Is your feature request related to a problem? Please describe:**

1. Set some filter conditions in the Slow-Query page;
![8464b274-214f-4625-a46e-a8d60dc95e40](https://user-images.githubusercontent.com/7499936/212234635-09ab7494-5ebc-4b3e-9b33-c9bf9e552e60.jpeg)

2. Then enter the detail page of any slow query and then exit;
<img width="698" alt="Glow Query Detail" src="https://user-images.githubusercontent.com/7499936/212234701-49d7a81d-82de-4953-b0d7-e3acc78a7c7a.png">

3. All existing filter conditions are cleared;
<img width="1072" alt="Pasted Graphic 7" src="https://user-images.githubusercontent.com/7499936/212234712-d2eeeb04-494f-4841-926e-2431755ae5de.png">

**Describe the feature you'd like:**

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

**Comment by shhdgit** [comment]
> /cc @shhdgit , it seems we need to use `navigate(-1)` to back?

yes, but we can't history back if a new tab comes from copy & paste.
